priv/catalog/156F64.yaml

```diff
@@ -14,4 +14,4 @@ facts:
     argument: totem.token
 expectations:
   - name: timeout
-    expect: corosync_token_timeout == 30000
+    expect: corosync_token_timeout == "30000"
```

Comment:
This is an interesting one, when the parts clicked the check failed :sweat_smile: 

The agent returns a string `"30000"`. while the expression containing a number.